uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/HRContactActionsMenu/Page/Inline/Actions.tpl

```diff
@@ -1,69 +1,70 @@
   {crmButton id="crm-contact-actions-link" href="#" icon="bars"}
-  {ts}Actions{/ts}
+    {ts}Actions{/ts}
   {/crmButton}
-  <div id="crm-contact-actions-list">
+  <div id="crm-contact-actions-list" class="crm_contact-actions">
     <div id="bootstrap-theme">
-      <div class="row">
-        <div class="col-md-4 crm_contact_action_menu__user_info">
-            {foreach from=$menu->getHighlightedPanelItems() item='group'}
-            <div class="row">
+      <div class="crm_contact-actions__body">
+        <div class="col-md-4 crm_contact-actions__body__user-info crm_contact-actions__user-info--margin">
+          {foreach from=$menu->getHighlightedPanelItems() item='group'}
+            <div class="col-md-12">
               <h3>{$group->getTitle()}</h3>
               {foreach from=$group->getItems() item='item'}
                 {$item->render()}
               {/foreach}
             </div>
-            {/foreach}
+          {/foreach}
+        </div>
+        <div class="col-md-8 crm_contact-actions__body__user-actions crm_contact-actions--bg-white">
+          {foreach from=$menu->getMainPanelItems() item='group'}
+            <div class="col-md-4">
+              <h3>{$group->getTitle()}</h3>
+              {foreach from=$group->getItems() item='item'}
+                {$item->render()}
+              {/foreach}
+            </div>
+          {/foreach}
+        </div>
+      <div>
+      <div class="crm_contact-actions__footer">
+        <div class="col-md-4 crm_contact-actions__footer__user-info">
           {if !empty($contactInfo.id)}
-            <div class="row crm_contact_action_menu__control_buttons_container">
-              <hr>
-              {if $userAccountDisabled}
-                <div>
-                  <span>ACCOUNT DISABLED</span>
-                </div>
-                <div class="col-md-6">
-                  <a href="/civicrm/contactactionsmenu/enableuseraccount?cid={$contactInfo.contact_id}">
-                    <button class="button__delete btn-warning">Enable User Account</button>
-                  </a>
-                </div>
-              {else}
-                <div class="col-md-6">
-                  <a href="/civicrm/contactactionsmenu/disableuseraccount?cid={$contactInfo.contact_id}">
-                    <button class="button__delete btn-warning">Disable User Account</button>
-                  </a>
-                </div>
-              {/if}
+            <div class="col-md-12">
+              <hr class="crm_contact-actions--dark-gray-blue">
+            </div>
+            {if $userAccountDisabled}
               <div class="col-md-6">
-                <a href="/civicrm/contactactionsmenu/deleteuseraccount?cid={$contactInfo.contact_id}">
-                  <button class="button__disable btn-danger">Delete User Account</button>
+                <a class="btn btn-warning crm_contact-actions__btn" href="/civicrm/contactactionsmenu/enableuseraccount?cid={$contactInfo.contact_id}">
```

Comment:
what's this `crm_contact-actions__btn` class for? Please add screenshot

Reply:
Currently test in button does not get wrapped as per button size. This class fixes this issue.
Form (issue): 
<img width="387" alt="screen shot 2018-01-24 at 1 46 49 pm" src="https://user-images.githubusercontent.com/6307362/35320771-13acc24e-010d-11e8-808e-7062b0b7b188.png">

To (fixed):
<img width="399" alt="screen shot 2018-01-24 at 1 52 26 pm" src="https://user-images.githubusercontent.com/6307362/35320994-dc29b178-010d-11e8-85f7-c64ddcbfac42.png">

Reply:
again, this is layout. Use the parent to define the layout of the children